test/parallel/test-cluster-net-listen-relative-path.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+const path = require('path');
+const fs = require('fs');
+
+// Ref:
+// https://github.com/nodejs/node/issues/16387
+// https://github.com/nodejs/node/pull/16749
```

Comment:
These refs can be removed during landing. They should be in the metadata, not in the comments.